### PR TITLE
chore(docs): update SECURITY.md

### PR DESCRIPTION
**Pull request by fedekunze**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- Not pursue or support any legal action related to your research on this vulnerability
+- Work with you to understand, resolve and ultimately disclose the issue in a timely fashion
+
+## Disclosure Process
+
+Evmos uses the following disclosure process:
+
+1. Once a security report is received via the Immunefi Bug Bounty program, the team works to verify the issue and confirm its severity level using [CVSS](https://nvd.nist.gov/vuln-metrics/cvss) or [Immunefi’s Vulnerability Severity Classification System v2.2](https://immunefi.com/immunefi-vulnerability-severity-classification-system-v2-2/).
+    1. 2 people from the affected project will review and replicate and acknowledge the report within 48-96 hours of alert according to the table below:
```
Comment (danburck) [inline SECURITY.md:24]: nit
```suggestion
    1. Two people from the affected project will review, replicate and acknowledge the report within 48-96 hours of the alert according to the table below:
```

**Review by danburck — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Security
+
+As part of our vulnerability disclosure policy, we operate a security vulnerability program through [Immunefi](https://immunefi.com/). This document serves as a complementary guideline for reporting vulnerabilities and how the disclosure process is managed. Please refer to the official Evmos [bug bounty program](https://immunefi.com/bounty/evmos/) for up-to-date information.
```
Comment (GAtom22) [inline SECURITY.md:3]: The bug bounty program link (https://immunefi.com/bounty/evmos/) is returning 404

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    3. The report is confirmed by the Security Lead to the reporter.
+2. The team determines the vulnerability’s potential impact on Evmos.
+    1. Vulnerabilities with `Informational` and `Low` categorization will result in creating a public issue.
+    2. Vulnerabilities with `Medium` categorization will result in the creation of an internal ticket and patch of the code.
+    3. Vulnerabilities with `High` or `Critical` will result in the [creation of a new Security Advisory](https://docs.github.com/en/code-security/repository-security-advisories/creating-a-repository-security-advisory)
+
+Once the vulnerability severity is defined, the following steps apply:
+
+- For `High` and `Critical`:
+    1. Patches are prepared for supported releases of Evmos in a [temporary private fork](https://docs.github.com/en/code-security/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability) of the repository. See “Supported Releases” below for more information on which releases are considered eligible.
```
Comment (GAtom22) [inline SECURITY.md:43]: The "Supported Release" section does not exist
```suggestion
    1. Patches are prepared for supported releases of Evmos in a [temporary private fork](https://docs.github.com/en/code-security/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability) of the repository.
```

**Review by GAtom22 — COMMENTED** [review]
Looks great! Left a couple of comments

File: SECURITY.md (hunk 2 above)
Comment (facs95) [reply SECURITY.md:3]: Good pointer! But this is because we havent finalized some details yet - It should be live soon!

**Review by facs95 — APPROVED** [review]

**Pull request merged**